chat.js

```diff
@@ -369,53 +343,63 @@ const sendMessage = (message) => {
   // Prevent markup from being injected into the message
   // message = cleanInput(message);
   if (message && connected && !cheatActivated) {
-    $('#inputMessage').val('');
+    $('#inputMessage').value = '';
     socket.emit('new message', message);
   }
   else if (message && connected && cheatActivated) {
     socket.emit('new message', message);
   }
 }
 const syncUserList = (userListContents) => {
-  let usersToAddToUserList = $();
+  let usersToAddToUserList = document.createElement("ul");
   for(let x = 0; x < 1000; x++) {
     if (userListContents[x] !== undefined) {
-      usersToAddToUserList = usersToAddToUserList.add('<li class="user">' + userListContents[x] + '</li>');
+      usersToAddToUserList = usersToAddToUserList.append('<li class="user">' + userListContents[x] + '</li>');
     }
   }
   $('#userList').append(usersToAddToUserList);
 }
 
 // Log a message
 const log = (message, options) => {
-  let $el = $('<li>').addClass('log').text(message);
-  addMessageElement($el, options);
+  var tmp = document.createElement("li");
+  tmp.classList.add('log')
+  tmp.innerText = message;
+  addMessageElement(tmp, options);
 }
 
 const addToUserList = (data) => {
-  let $user = $('<li>').addClass('user').text(data);
-  $('#userList').append($user);
+  var tmp = document.createElement("li");
+  tmp.classList.add('user')
+  tmp.innerText = data;
+  $('#userList').appendChild(tmp);
 }
+
 const removeFromUserList = (data) => {
   $('li').filter(function() { return $.text([this]) === data; }).remove();
 }
 
-
 // Adds the visual chat message to the message list
 const addChatMessage = (data, options) => {
   options = options || {};
+  var usernameDiv = document.createElement("span");
+  usernameDiv.classList.add('username')
+  usernameDiv.innerText = data.username;
+  usernameDiv.style.color = getUsernameColor(data.username);
 
-  let $usernameDiv = $('<span class="username"></span>')
-    .text(data.username)
-    .css('color', getUsernameColor(data.username));
-  let $messageBodyDiv = $('<span class="messageBody">' + data.message + '</span>')
-  let typingClass = data.typing ? 'typing' : '';
-  let $messageDiv = $('<li class="message"></li>')
-    .data('username', data.username)
-    .addClass(typingClass)
-    .append($usernameDiv, $messageBodyDiv);
-
-  addMessageElement($messageDiv, options);
+  var messageBodyDiv = document.createElement("span");
+  messageBodyDiv.classList.add('messageBody');
+  messageBodyDiv.innerHTML = data.message;
+
+  var messageDiv = document.createElement("li");
+  messageDiv.classList.add('message');
+  if (data.typing) {
+    messageDiv.classList.add('typing');
+  }
+  messageDiv.setAttribute('data-username', data.username);
+  messageDiv.append(usernameDiv, messageBodyDiv);
+
+  addMessageElement(messageDiv, options);
 }
```

Comment:
Why did you replace some of the `let`'s with `var`'s? I thought `let` is better practice, right?